codesystems/CodeSystem-England-EPSIssueCode.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure about the xml tag, as remaining files doesn't  have

Reply:
Many files do have it, don't think that's a problem